commands/createfixpullrequests.go

```diff
@@ -143,9 +143,9 @@ func (cfp *CreateFixPullRequestsCmd) fixVulnerablePackages(fixVersionsMap map[st
 	return
 }
 
-func (cfp *CreateFixPullRequestsCmd) fixIssuesSeparatePRs(fixVersionsMap map[string]*utils.FixVersionInfo) (err error) {
-	for impactedPackage, fixVersionInfo := range fixVersionsMap {
-		if err = cfp.fixSinglePackageAndCreatePR(impactedPackage, fixVersionInfo); err != nil {
+func (cfp *CreateFixPullRequestsCmd) fixIssuesSeparatePRs(fixVersionsMap map[string]*utils.FixDetails) (err error) {
+	for _, fixVersionInfo := range fixVersionsMap {
+		if err = cfp.fixSinglePackageAndCreatePR(fixVersionInfo); err != nil {
 			log.Warn(err)
```

Comment:
Not sure thsi should be a warning log message. This should probably an error.
I think however that it is best to accumulate these error messages into a single error struct, thst the function will return at the end. If no error occuured, the function should return a nil error of course.